src/content/extensions/multipleroutes/src/options.js

```diff
@@ -0,0 +1,49 @@
+// Saves options.
+function save_options() {
+  var multiRoutes = document.getElementById('multiRoutes').checked;
+  var nonProxiedUdp = document.getElementById('nonProxiedUdp').checked;
+  chrome.privacy.network.webRTCMultipleRoutesEnabled.set({
+    'value': multiRoutes
+  });
+  try {
+    chrome.privacy.network.webRTCNonProxiedUdpEnabled.set({
+      'value': nonProxiedUdp
+    });
+  } catch (err) {
+    document.getElementById('nonProxiedUdp').checked = false;
+    document.getElementById('nonProxiedUdp').disabled = true;
+  }
```

Comment:
If something goes wrong, shouldn't we set it back to what is was?

document.getElementById('nonProxiedUdp').checked = nonProxiedUdp;